styles/header/header.module.scss

```diff
@@ -0,0 +1,89 @@
+.header {
+  background: url('../../public/img/roses.png') no-repeat center center / cover;
+  color: var(--white-color);
+  position: relative;
+  font-weight: var(--font-light);
+  padding: 25px 25px 22px 25px;
+
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: rgba(0, 0, 0, 0.2);
+    z-index: 1;
+  }
+
+  .content {
+    height: 500px;
+    position: relative;
+    z-index: 2;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    .top {
+      display: flex;
+      justify-content: space-between;
+      align-items: flex-start;
+
+      .logo {
+        display: flex;
+
+        .gallery {
+          font-size: 15px;
+          line-height: 1.3;
+        }
+      }
+
+      // Mixin for icons
+      @mixin icon-before($width, $height, $image-url) {
```

Comment:
To use the mask you need to remake the mixin like this
`      @mixin icon-before($width, $height, $image-url) {
        position: absolute;
        content: '';
        -webkit-mask: url($image-url) no-repeat 50% 50%;
        mask: url($image-url) no-repeat 50% 50%;
        background: var(--white-color);
        height: $height;
        width: $width;
        top: 0;
        left: 0;
      }`